### ISSUE TITLE
Support for Laravel 5.8

### ISSUE THREAD

**Issue by sroseman** [acoustep/entrust-gui#41]
Will support for Laravel 5.8 be released soon? Basically the biggest annoyance is that I am going in and changing all fire() to dispatch() and hoping that my team reads my readme about doing that and then never doing a composer update.

**Comment by acoustep** [comment]
Hi @sroseman , thanks for trying out the package!

I've made a start on a 5.8 branch which you can now see here https://github.com/acoustep/entrust-gui/tree/5.8

I've updated most references and dependencies, however I'm currently getting an issue with the fire event (even though I've renamed them). 

I'm short on time at the moment so I can't investigate further right now - Any chance you could let me know what changes you've made and in what files?

**Comment by sroseman** [comment]
@acoustep , I had to change fire to dispatch within ManyToManyGateway.php, UserGateway.php, and DeleteModelTrait.php within your vendor files, and I needed to change it in ValidatingObserver.php within the watson vendor folder. 

I used this package as a proof of concept for our team, so I haven't exhausted my testing of seeing if that fixed the issues, but I believe I tested basic CRUD operations with no issues.

**Comment by Shohan494** [comment]
@acoustep,@sroseman hello guys, sorry to bother but any update on this? facing the same issue that's why thought of giving a ping.. Thanks

**Comment by acoustep** [comment]
Hi @sroseman @Shohan494, 

I have just pushed an update to the 5.8 branch which should now work https://github.com/acoustep/entrust-gui/tree/5.8 - You will need to re-check the install instructions, as there are a couple of extra steps/hiccups.

For a little extra context, there is an issue with one of the dependencies (Esensi/model) which depends on an older version of watson/validating which does not support 5.8. Unfortunately Esensi have not updated this yet. 

I have tested forcing the watson/validating package from 2.0.1 to 3.2.0 to fix the problem and I've had no issues so far. I recommend testing this out and seeing if you have any problems.

There are gradually more issues cropping up due to a couple of dependencies which is making me itch for a rewrite - however, for now this should hopefully solve your problems.
